.gitignore

```diff
@@ -3,3 +3,4 @@ examples/*/*
 coverage.out
 tags
 *.swp
+**/.idea
```

Comment:
What's that one?

Reply:
I'm working with Goland and to prevent adding it's internal meta to the repo I added this line to ``.gitignore``.

Reply:
Can you split this change into its own commit?

Reply:
I also can remove it in the next commit if that's ok for you.